### DISCUSSION TITLE
Linking Rust library as C code

### DISCUSSION THREAD

**Discussion by amunra** [corrosion-rs/corrosion#189] [General]
Hello,

I've ran into issues where corrosion adds a dependency to the C++ standard library. In my case, this is unwanted.
I've raised a draft review with how I'm tackling this, but I'd welcome feedback: https://github.com/corrosion-rs/corrosion/pull/188. Am I roughly moving in the right direction?

Thanks,
Adam

**Comment by jschwe** [comment]
I already commented on your PR, but as a general comment: Have you tried the `NO_STD` argument for `corrosion_import_crate`? If you don't require any C/C++ standard library functions, then this may already be sufficient for you.

**Reply by amunra** [reply]
Good point, but not really an option for my use case.
